src/atoms/second_order_cone/quadform.jl

```diff
@@ -1,6 +1,6 @@
 export quadform
 
-function is_symmetric(A::Matrix)
+function is_symmetric(A::AbstractArray)
   TOLERANCE = 1e-6
   return all(A - A' .< TOLERANCE)
```

Comment:
Cool, just merged a fix to use issym from Base instead.

On Sun, Jul 10, 2016 at 12:03 PM, Tony Kelman <notifications@github.com>
wrote:

> In src/atoms/second_order_cone/quadform.jl
> <https://github.com/JuliaOpt/Convex.jl/pull/149#discussion_r70184255>:
>
> > @@ -1,6 +1,6 @@
> >  export quadform
> >
> > -function is_symmetric(A::Matrix)
> > +function is_symmetric(A::AbstractArray)
> >    TOLERANCE = 1e-6
> >    return all(A - A' .< TOLERANCE)
>
> there's also an issym in base that is likely smart about short-circuiting
> etc
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaOpt/Convex.jl/pull/149/files/10f5913c6183410633782fd5aa3faf30155e1109#r70184255>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAyp9FV2EU9zvq9aXsEpb9gPQB-CZKJCks5qURfWgaJpZM4JIeLH>
> .
>



-- 
Madeleine Udell
Assistant Professor, Operations Research and Information Engineering
Cornell University
*https://courses2.cit.cornell.edu/mru8
<https://courses2.cit.cornell.edu/mru8>*
(415) 729-4115